string platooning_state
uint32 platoon_id
float32 ipd
float32 pd
uint32 vehicle_id
bool i_am_FV
bool i_am_LV